operations/onboarding.md

```diff
@@ -36,12 +36,33 @@ TOC members rotate meeting facilitation as often as we can to ensure the TOC Cha
 
 In the course of performing our duties, TOC members are expected to:
 
+* Lead or participate in at least 1 Project Due Diligence that is brought to TOC internal review status
+    * Some Due Diligence efforts may result in the TOC internal review stating the project is not ready after adopter interviews, these are still considered as meeting this expectation. Lightweight reviews/triage would not.
```

Comment:
+1 to this. Thanks!

Reply:
Am I right in reading this as stating the number of concurrent DDs a TOC member should have in flight, but not commenting at all on the length of time that a DD is expected to take?  

I'm not sure that it addresses "throughput" if so.

Reply:
throughput is probably not the right word if you're looking for the volume of due diligences that move through the moving levels process. These are metrics we've asked staff to begin looking at. this is should ensure each TOC member is aware that minimally they must lead or participate in 1 DD that reaches a state of decision post-evaluation. I cannot define the time a DD is expected to take because there are a number of factors that directly influence timeliness outside of a TOC member's control. We attempt to provide an estimate, but ultimately availability of all stakeholders (the project, the adopters, and the TOC member) are the most challenging impacts to time in review. We are working to get that information however.

Reply:
i've added the clarification for 1 DD per year minimum.